src/library_webgpu.js

```diff
@@ -1522,26 +1522,26 @@ var LibraryWebGPU = {
       #if ASSERTIONS
           assert(depthLoadOpInt === {{{ gpu.LoadOp.Clear }}} || depthLoadOpInt === {{{ gpu.LoadOp.Load }}});
       #endif
```

Comment:
This is also now allowed to be undefined. I think you can just delete the asserts (and `depthLoadOpInt` and `stencilLoadOpInt`).

Reply:
Yep. And loadOp, storeOp of color attachment are required and should add this assertion.